koa-crud/src/data-access/mongoose/vendors/actions.ts

```diff
@@ -12,11 +12,11 @@ const actions = ({ Vendor }): VendorStore => {
   }
 
   async function selectAllVendors() {
-    return Vendor.find().lean();
+    return Vendor.find().lean({virtuals: true});
   }
 
   async function selectOneVendorByFilters(filters) {
-    return Vendor.findOne({ ...filters }).lean();
+    return Vendor.findOne({ ...filters }).lean({virtuals: true});
```

Comment:
If you are only using `filters` here
```ts
{ ...filters }
```

You can also apply it directly
```ts
    return Vendor.findOne(filters).lean({virtuals: true});

```

Reply:
noted sir